nestkernel/simulation_manager.cpp

```diff
@@ -102,6 +102,7 @@ nest::SimulationManager::set_status( const DictionaryDatum& d )
   TimeConverter time_converter;
 
   double time;
+  double eps = 1e-10;
```

Comment:
Please make this `const` and give it a more expressive name (ALL_CAPS). Could you reduce the value even further, around 1e-14?